Makefile

```diff
@@ -523,6 +529,8 @@ PREVIOUS_TAG ?= $(shell git tag -l | grep -E "^v[0-9]+\.[0-9]+\.[0-9]+$$" | sort
 RELEASE_ALIAS_TAG := $(PULL_BASE_REF)
 RELEASE_DIR := out
 RELEASE_NOTES_DIR := _releasenotes
+USER_FORK ?= $(shell git config --get remote.origin.url | cut -d/ -f4)
```

Comment:
This assumes the user should have well-known names for remotes.
Should we document it somewhere under https://cluster-api.sigs.k8s.io/developer/guide.html or in https://cluster-api.sigs.k8s.io/contributing ?
Another alternative is to link https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md from the same documents

Reply:
Another alternative is to ask explicitly for the fork when calling make promote-images

Reply:
I used `?=` so you can always override it with your own USER_FORK variable, this is just a smart default that should fit most people. Since this command will only be used by maintainers, I expect each one of us will have our own flow that works for us.